server/model/agent.go

```diff
@@ -14,12 +14,19 @@
 
 package model
 
+import (
+	"encoding/base32"
+	"fmt"
+
+	"github.com/gorilla/securecookie"
+)
+
 type Agent struct {
 	ID          int64  `json:"id"            xorm:"pk autoincr 'id'"`
 	Created     int64  `json:"created"       xorm:"created"`
 	Updated     int64  `json:"updated"       xorm:"updated"`
 	Name        string `json:"name"          xorm:"name"`
-	OwnerID     int64  `json:"owner_id"      xorm:"'owner_id'"`
+	OwnerID     int64  `json:"owner_id"      xorm:"INDEX 'owner_id'"`
```

Comment:
Doesn't really seem that we query by the `owner_id` so the index should not be necessary. In general the `owner_id` might be redundant to `org_id` & `repo_id` now.